internal/photoprism/index.go

```diff
@@ -52,6 +52,11 @@ func (ind *Index) thumbPath() string {
 	return ind.conf.ThumbPath()
 }
 
+// IsRunning checks whether the indexer is currently running
+func (ind *Index) IsRunning() bool {
```

Comment:
I'd rename `IsRunning()` to `Busy()` to avoid different names for the same flag.